typescript/ecs/ecs-service-with-queue-processing/index.ts

```diff
@@ -0,0 +1,36 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import ecs = require('@aws-cdk/aws-ecs');
+import ecs_patterns = require('@aws-cdk/aws-ecs-patterns');
+import cdk = require('@aws-cdk/core');
+
+class NihaoECS extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // For better iteration speed, it might make sense to put this VPC into
+    // a separate stack and import it here. We then have two stacks to
+    // deploy, but VPC creation is slow so we'll only have to do that once
+    // and can iterate quickly on consuming stacks. Not doing that for now.
+    const vpc = new ec2.Vpc(this, 'MyVpc', { maxAzs: 2 });
+    const cluster = new ecs.Cluster(this, 'Ec2Cluster', { vpc });
+    cluster.addCapacity('DefaultAutoScalingGroup', {
+      instanceType: ec2.InstanceType.of(ec2.InstanceClass.T2, ec2.InstanceSize.MICRO)
+    });
+
+     // Instantiate ECS Service with cluster and image
+    const ecsService = new ecs_patterns.QueueProcessingEc2Service(this, "Ec2Service", {
+      cluster,
+      memoryLimitMiB: 512,
+      image: ecs.ContainerImage.fromRegistry("amazon/amazon-ecs-sample"),
```

Comment:
This is not an image that does anything with the queue contents, right? 

What is the value of this example exactly?

Reply:
Thanks for the review. It provides an example of how to create a queue processing service using the high level construct defined in the ecs pattern module. If that doesn't seem of any value to you, then I can close the pull request.